src/components/Table/index.tsx

```diff
@@ -404,9 +404,12 @@ class Table<T> extends React.Component<TableProps<T>, TableState<T>> {
 
 		const sortedData = this.sortData(items).slice(lowerBound, upperBound);
 
+		const shouldShowPaper =
+			!!usePager && totalItems > _itemsPerpage && totalItems > 0;
```

Comment:
It might be better to show the pager when there are items, so as they know how many items are in the table, and also to have a more consistent experience. I think we can hide it only where there are no items at all, as then it doesn't make much sense.